devices.js

```diff
@@ -8875,6 +8875,30 @@ const devices = [
             });
         },
     },
+    
+    // Develco 
+    {
+        zigbeeModel: ['SMSZB-120'],
+        model: 'SMSZB-120',
+        vendor: 'Develco Products A/S',
+        description: 'Smoke detector with siren',
+        supports: 'smoke, warning, temperature',
+        fromZigbee: [fz.temperature, fz.bitron_battery_att_report, fz.develco_smoke, fz.ignore_basic_report, fz.ignore_genOta],
```

Comment:
Instead of `fz.develco_smoke` can you use `fz.ias_smoke_alarm_1`?

Reply:
I will test and come back to you

Reply:
Just tested it. Yes, it works, but there is no "enrolled" in fz.ias_smoke_alarm_1 so I cannot see if the sensor enrolls

Reply:
Feel free to add enrolled (https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/fromZigbee.js#L572)